libfwupd/fwupd-enums.c

```diff
@@ -848,6 +848,8 @@ fwupd_version_format_from_string(const gchar *str)
 		return FWUPD_VERSION_FORMAT_SURFACE;
 	if (g_strcmp0(str, "dell-bios") == 0)
 		return FWUPD_VERSION_FORMAT_DELL_BIOS;
+	if (g_strcmp0(str, "dell-bios-modern") == 0)
```

Comment:
I don't really like the name modern. I don't think it's going to age very well.

Proposals:
* dell-bios-2024
* dell-bios-aabbcc

Reply:
I was pondering for 2024 at the beginning, though aabbcc looks clear and straight